.env.example

```
@@ -14,6 +14,17 @@ NEXT_PUBLIC_APP_URL=http://localhost:3000
 # Any non-empty random string for local dev: openssl rand -hex 16
 CRON_SECRET=generate-a-random-secret
 
+# Receipt hunt (nightly matcher, 05:30 UTC): comma-separated company ids the
+# hunt may stage proposals for. Unset means it runs for nobody, so enabling it
+# is always a deliberate act rather than a side effect of deploying.
+# Model used to resolve bank descriptors to merchants and to decide which mail
+# is the receipt for which purchase. Falls back to BEDROCK_MODEL_ID.
+RECEIPT_HUNT_MODEL_ID=
+# Floor for accepting the model's pairing. Every proposal is human-reviewed, so
+# this trades recall against review effort, not against correctness.
+RECEIPT_HUNT_MIN_CONFIDENCE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RECEIPT_HUNT_MIN_CONFIDENCE key should go before the RECEIPT_HUNT_MODEL_ID key

(UnorderedKey)